hashintel/hash
# Implement creating entity type functionality

## 🌟 What is the purpose of this PR?




Add ability to create an entity type to the entity type editor

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1202953520344821/1203109490608734/f) _(internal)_

## 🚫 Blocked by




N/A

## 🔍 What does this change?




- Update design of create entity screen to more closely match design
- New `useLoggedInUser` which redirects to login if not logged in
- Updates URLs to `/@shortname/types/new/entity-type` and `/@shortname/types/entity-type/slug` and updates ontology chips
- Makes submitting the create form work, and makes it redirect to the view page
- Make the view page display the entity type title

## 📜 Does this require a change to the docs?



- The docs for x need updating to explain that y

## ⚠️ Known issues



## 🐾 Next steps



Validation

## 🛡 What tests cover this?



N/A

## ❓ How to test this?



1. Create an entity type
2. See that you're redirected to the entity type, and the title displays correctly
